spec/features/navigation/navigation_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'Site Navigation' do
+  before :each do
+
+    visit '/'
+
```

Comment:
For this test I just used our same setup from the new spec to have us logged in as a user, the story did not say where to start out so I thought this was as good as any place.